both/routes.js

```diff
@@ -25,27 +25,26 @@ if(Meteor.isClient) {
     return Router.current().route.getName() == routeName;
   });
 
-  Template.registerHelper("refreshTitle", function() {
-    var title = "live.cubing.net";
+  Template.registerHelper("documentTitle", function() {
```

Comment:
I prefer the name `refreshTitle` here as it sounds like something with a side effect. `documentTitle` sounds like it should return a string and do nothing else.